schema/patient.js

```diff
@@ -17,6 +17,7 @@ const patientSchema = `
   extend type Query {
     patient(id: ID!): Patient
     patients: [Patient]
+    patientsByCCPId(collectionPointId: ID!): [Patient]
```

Comment:
do we have a pattern of calling the id `collectionPointId` already (like with the full entity name)? if not i would prefer `ccpId`
```suggestion
    patientsByCcp(collectionPointId: ID!): [Patient]
```

Reply:
We do, check line 29 and line 44